Description: Do not not install into ~/.local
 setup.py tries to be smart about installing into $XDG_DATA_HOME, but this is
 never the right location for building a system package.
Author: Tristan Seligmann <mithrandi@debian.org>
Forwarded: not-needed
Last-Update: 2013-11-28
--- a/setup.py
+++ b/setup.py
@@ -16,8 +16,6 @@
     sys.exit("Error: Electrum requires Python version >= 2.6.0...")
 
 usr_share = '/usr/share'
-if not os.access(usr_share, os.W_OK):
-    usr_share = os.getenv("XDG_DATA_HOME", os.path.join(os.getenv("HOME"), ".local", "share"))
 
 data_files = []
 if (len(sys.argv) > 1 and (sys.argv[1] == "sdist")) or (platform.system() != 'Windows' and platform.system() != 'Darwin'):
